front50-core/src/main/java/com/netflix/spinnaker/front50/config/Front50CoreConfiguration.java

```diff
@@ -38,4 +42,14 @@ public RestTemplate restTemplate() {
   public Namer<?> namer() {
     return new FriggaReflectiveNamer();
   }
+
+  @Bean
+  Jackson2ObjectMapperBuilderCustomizer defaultObjectMapperCustomizer(List<Module> modules) {
+    return jacksonObjectMapperBuilder -> {
+      modules.addAll(List.of(new Front50ApiModule()));
+      jacksonObjectMapperBuilder.failOnEmptyBeans(false);
```

Comment:
Are these two flags needed for your changes? I would try to leave them out if possible.